lib/ProMotion/iap.rb

```diff
@@ -124,50 +164,6 @@ def transaction_product_id(transaction)
       transaction.respond_to?(:payment) ? transaction.payment.productIdentifier : "all"
     end
 
-    public
```

Comment:
I could be swayed on this, but I liked grouping the delegate methods at the end.